src/chat_statistics/stats.py

```diff
@@ -0,0 +1,81 @@
+import json
+from pathlib import Path
+from typing import Union
+
+import arabic_reshaper
+from bidi.algorithm import get_display
+from hazm import Normalizer, word_tokenize
+from loguru import logger
+from src.data import DATA_DIR
+from wordcloud import WordCloud
+
+
+class ChatStatistics:
+    """Generataes chat statistics from a telehram chat json file
+    """
+    def __init__(self, chat_json: Union[str, Path]):
+        """
+        :param chat_json:  path to telegram export json file
+        """
+        # Load chata_data
+        logger.info(f"Loading chat data from {chat_json}")
+        with open(chat_json) as f:
+            self.chat_data = json.load(f)
+
+        self.normalizer = Normalizer()
+
+        # # Load stop_words
+        logger.info(f"Loading stopwords from {DATA_DIR / 'stop_words.txt'}")
+        # with open (str(DATA_DIR / 'stop_words.txt')) as sws:
+        #     stop_words = sws.read().split('\n')
+        #     self.stop_words = list(map(self.normalizer.normalize, stop_words)
+
+        stop_words = open(str(DATA_DIR / 'stop_words.txt')).readlines()
+        stop_words = list(map(str.strip, stop_words))
+        self.stop_words = list(map(self.normalizer.normalize, stop_words))
+
+    def generate_word_cloud(
+            self, output_dir: Union[str, Path],
+            width: int = 800, height: int = 600,
+            max_font_size: int = 250
+            ):
```

Comment:
### Better to _add_ other **optional wordcloud arguments** such as 'background color'. 
```suggestion
    def generate_word_cloud(
            self, output_dir: Union[str, Path],
            width: int = 800, height: int = 600,
            max_font_size: int = 250,
            background_color='white',
            ):
```